tests/test_panorama/test_application_instance.py

```diff
@@ -0,0 +1,394 @@
+import datetime
+from unittest import SkipTest
+from unittest.mock import ANY
+
+import boto3
+from dateutil.tz import tzutc
+from freezegun import freeze_time
+
+from moto import mock_aws, settings
+
+GIVEN_DEVICE_NAME = "not-a-device-name"
+PROVISION_DEVICE_PARAMS = {
+    "Description": "not a device description",
+    "Name": GIVEN_DEVICE_NAME,
+    "NetworkingConfiguration": {
+        "Ethernet0": {
+            "ConnectionType": "STATIC_IP",
+            "StaticIpConnectionInfo": {
+                "DefaultGateway": "192.168.1.1",
+                "Dns": [
+                    "8.8.8.8",
+                ],
+                "IpAddress": "192.168.1.10",
+                "Mask": "255.255.255.0",
+            },
+        },
+        "Ethernet1": {
+            "ConnectionType": "dhcp",
+        },
+        "Ntp": {
+            "NtpServers": [
+                "0.pool.ntp.org",
+                "1.pool.ntp.org",
+                "0.fr.pool.ntp.org",
+            ]
+        },
+    },
+    "Tags": {"Key": "test-key", "Value": "test-value"},
+}
+
+
+@mock_aws
+def test_create_application_instance() -> None:
+    # Given
+    panorama_client = boto3.client("panorama")
+    response_device_creation = panorama_client.provision_device(
+        **PROVISION_DEVICE_PARAMS
+    )
+
+    # When
+    response = panorama_client.create_application_instance(
+        DefaultRuntimeContextDevice=response_device_creation["DeviceId"],
+        Description="not a description",
+        ManifestOverridesPayload={
+            "PayloadData": "dumped_payload_data",
+        },
+        ManifestPayload={
+            "PayloadData": "dumped_manifest_payload_data",
+        },
+        Name="not-a-name",
+        RuntimeRoleArn="not-an-arn",
+        Tags={
+            "Key": "value",
+        },
+    )
+    # Then
+    assert isinstance(response["ApplicationInstanceId"], str)
+    assert response["ApplicationInstanceId"].startswith("applicationInstance-")
+    assert len(response["ApplicationInstanceId"]) == 42
+
+
+@mock_aws
+def test_describe_application_instance() -> None:
+    # Given
+    panorama_client = boto3.client("panorama")
```

Comment:
```suggestion
    panorama_client = boto3.client("panorama", "eu-west-1")
```

We assume that the ARN starts with `arn:aws:panorama:eu-west-1..` - but that is only the case if the region is set to `eu-west-1`. If the region is not explicitly configured, our CI assumes it is set in `us-east-1`, which results in a different ARN.

This will need to be changed in the other tests as well.

Reply:
6de03bd1c02cf1d20ddfae1bff52e7373ccbe08f